book/scheduling-and-threading.md

```diff
@@ -174,18 +174,29 @@ Asynchronous rendering
 ======================
 
 In order to separate rendering from other tasks and make it into a proper
-pipeline, we'll need to make it asynchronous. Instead of updating rendering
-right away, we'll set *dirty bits* indicating that a particular part of the
-pipeline needs updating. Then when the pipeline is run, we'll run the parts
+pipeline, let's make it asynchronous. Instead of updating rendering right away,
+the browser should set *dirty bits* indicating that a particular part of the
+pipeline needs updating. Then when the pipeline is run, it will run the parts
 indicated by the dirty bits. We'll also need some way of
 *scheduling* the rendering pipeline to be updated at a given time in the
 future.
 
 Let's start with how to schedule the update, via a new `set_timeout` function.
-This function will run a callback at a specified time in the future.
-You can do that by starting a new [Python thread][python-thread] via the
-`threading.Timer` class, which takes two parameters: a time delta from now, and
-a function to call when that time expires.
+This function will run a callback at a specified time in the future. You can do
+that by starting a new [Python thread][python-thread] via the `threading.Timer`
+class, which takes two parameters: a time delta from now, and a function to
+call when that time expires. It will start a new thread and call that
+function *on that thread*[^thread-timer] at the desired time. When the
+function completes, the timer thread is automatically ended.
+
+[^thread-timer]: This is convenient because it enables the timer to
+execute at about the time desired---even if the current thread is busy at that
+time---but, as we'll see, leads to some complications if you really want the
+function to be called on the same thread as the one calling `set_timeout`.
+But in any case, it wouldn't make any sense to run the task on the "current"
```

Comment:
I found this footnote really confusing. The actual point it's making is pretty clear, so we should frame it positively (here's what's true) instead of negatively (here's why that other belief is false).

Reply:
Fixed in another PR.